src/openrct2-cli/CMakeLists.txt

```diff
@@ -30,5 +30,5 @@ if (FORCE32)
     set(TARGET_M "-m32")
 endif ()
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu11 ${TARGET_M}")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++14 ${TARGET_M}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++17 ${TARGET_M}")
```

Comment:
I think these override each other by being included from the top-level makefile, so perhaps some refactoring is needed here.